miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/service/integration/jira/JiraIssueManager.java

```diff
@@ -58,55 +66,52 @@
  */
 @Component
 public class JiraIssueManager implements IssueTrackerManager {
+
+  private static final String restApiUrl = "/rest/api/";
+  private static final String jiraRestApiVersion = "2";
+
+  private final DateFormat ISO8601_FORMAT = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "ISO8601_FORMAT" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00116)